src/benchmarks/micro/runtime/Linq/Linq.cs

```diff
@@ -327,7 +320,6 @@ public bool Order00LinqQueryX()
             var productsInPriceOrder = from prod in products orderby prod.UnitPrice descending select prod;
             int count = productsInPriceOrder.Count();
             medianPricedProduct = productsInPriceOrder.ElementAt<Product>(count / 2);
-            Escape(medianPricedProduct);
```

Comment:
Any reason why this has been removed?

Reply:
it's simply not needed, as the variable is used to calculate the result of the method, so it won't get dead code eliminated:

https://github.com/dotnet/performance/blob/413d1aa6aed0d05e69bff2de8ff9b59513779679/src/benchmarks/micro/runtime/Linq/Linq.cs#L325

Reply:
Ah, true. In this loop, we don't have any dependency on `i` so everything can be hoisted out. But I believe we don't do it because it has function calls. Anyways, nothing actionable here.